array-manipulation-1/src/head.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-unused-vars */
+function head(array) {
+  if (array === null) {
+    return undefined;
+  }
```

Comment:
No need to check if the array is null, it won't ever be null because we never pass in null.